.travis.yml

```diff
@@ -0,0 +1,56 @@
+---
+
+group: stable
+dist: trusty
+
+language: python
+python: "2.7"
+
+branches:
+  only:
+    - develop
+    - master
+
+env:
+  - ANSIBLE_INSTALL_VERSION=2.0.2.0
+  - ANSIBLE_INSTALL_VERSION=2.1.6.0
+  - ANSIBLE_INSTALL_VERSION=2.2.3.0
+  - ANSIBLE_INSTALL_VERSION=2.3.2.0
+  - ANSIBLE_INSTALL_VERSION=2.4.0.0
+
+before_install:
+  # Make sure everything's up to date.
+  - sudo apt-get update -qq
+
+install:
+  # Install Ansible.
+  - pip install ansible==${ANSIBLE_INSTALL_VERSION} ansible-lint
+
+script:
+  # Install Role Dependencies
```

Comment:
Can you take a look at the travis.yml file from the scaffold branch in the Sansible project? This looks a bit out of date.

Reply:
Done.